src/DynamicEvaluator/Presentation/Checker/BrokenPhpFilesChecker/ViolationBodyMessageBuilder.php

```diff
@@ -26,7 +26,7 @@ class ViolationBodyMessageBuilder implements CheckerViolationMessageBuilderInter
     /**
      * @var string
      */
-    protected const ERROR_MESSAGE = 'Issues in php files after "composer require" command';
+    protected const ERROR_MESSAGE = 'PHP classes that became not compatible with Spyker Release';
```

Comment:
Spyker -> Spryker

PHP classes that are no longer compatible with the Spryker Release.